src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/GeneOverview.scss

```diff
@@ -41,56 +43,42 @@
 
 .accordionContainer {
   width: 100%;
-}
-
-.entityViewerAccordion {
-  border: none;
-  margin-top: 30px;
-}
-
-.entityViewerAccordionHeader {
-  background-color: $light-grey;
-  border-radius: 5px;
-}
 
-.entityViewerAccordionButton {
-  color: $blue;
-  padding: 5px 0 5px 20px;
-
-  &:before,
-  &:after {
-    margin-top: -3px;
+  .entityViewerAccordion {
+    border: none;
+    margin-top: 30px;
   }
-
-  &Disabled {
-    color: $grey;
+  
+  .entityViewerAccordionHeader {
+    background-color: $light-grey;
+    border-radius: 5px;
+  }
+  
+    .entityViewerAccordionButton {
+    color: $blue;
+    padding: 5px 0 5px 20px;
+  
+    &:before,
+    &:after {
+      margin-top: -3px;
+    }
+  
+    &Disabled {
+      color: $grey;
+    }
+  }
```

Comment:
- do we need all these classes to be nested in the `accordionContainer`?
- something wrong with the indentation of `.entityViewerAccordionButton`
- the indentation inside the `accordionContainer` class is currently weird (what's going on with `sequenceAccordion` and `accordionContentTitle` classes and that closing curly brace in the bottom)?

<img src="https://user-images.githubusercontent.com/6834224/104619385-b8713c80-5685-11eb-8436-ff8477590707.png" width="500px" />

Reply:
I was just being lazy there thinking that the post commit hook will take care of it!
That's all sorted now. I have also removed some more classes that are not used at the moment.